tests/contracts/test_deposit.py

```diff
@@ -88,7 +106,10 @@ def test_reciept_tree(registration_contract, w3, assert_tx_failed):
         data = amount_bytes8 + timestamp_bytes8 + deposit_input
         leaf_nodes.append(w3.sha3(data))
         root = compute_merkle_root(w3, leaf_nodes)
-        assert registration_contract.functions.get_receipt_root().call() == root
+        assert registration_contract.functions.get_deposit_root().call() == root
+        index = randint(0, i)
```

Comment:
I tested with this setting and observe that the time it take to run the test did increase significantly but strangely I did not see any additional tests with different random number set performed and on top of that the random numbers generated in each test run are the same. 

Both of these problems may require further investigation into `hypothesis`. IMO it might not worth the effort to replace `random` with `hypothesis.strategies.random` here.